build/AzurePipelinesTemplates/MUX-BuildDevProject-Steps.yml

```diff
@@ -6,6 +6,8 @@ steps:
       appxPackageDir: $(appxPackageDir)
       buildOutputDir: $(buildOutputDir)
       publishDir: $(publishDir)
+      ${{ if ne(variables['System.CollectionUri'],'https://dev.azure.com/ms/') }}:
+        useInsiderSDK: true
```

Comment:
I don't think this will have the intended effect. The "${{ if ...}" syntax is a template expression which happens at "compile" time, but I think that variable will only get set at some point after that. I think. I find this aspect of Azure Pipelines very confusing.

Reply:
I launched the WinUI-Public-MUX-CI pipeline to see what happens...

Reply:
WinUI2-MUX-CI is using 10.0.21307.0 and WinUI-Public-MUX-CI is using 10.0.18362.0, so it seems to work.

Reply:
Ok cool. I told you I found it confusing! :)

Reply:
This is gone now.